src/main/java/com/sequenceiq/cloudbreak/service/stack/connector/aws/AwsConnector.java

```diff
@@ -218,6 +216,27 @@ public boolean removeInstances(Stack stack, Set<String> instanceIds, String host
         return true;
     }
 
+    @Override
+    public boolean updateAllowedSubnets(Stack stack, String userData) {
+        long stackId = stack.getId();
+        String cFStackName = cfStackUtil.getCfStackName(stack);
+        UpdateStackRequest updateStackRequest = new UpdateStackRequest()
+                .withStackName(cFStackName)
+                .withTemplateBody(cfTemplateBuilder.build(stack, "templates/aws-cf-stack.ftl",
+                        spotPriceNeeded(stack.getInstanceGroups()), stack.getInstanceGroupsAsList()))
+                .withParameters(getStackParameters(stack, userData, (AwsCredential) stack.getCredential(), stack.getName()));
+        AmazonCloudFormationClient cloudFormationClient = awsStackUtil.createCloudFormationClient(stack);
+        cloudFormationClient.updateStack(updateStackRequest);
+        List<StackStatus> errorStatuses = Arrays.asList(UPDATE_ROLLBACK_COMPLETE, UPDATE_ROLLBACK_FAILED);
+        CloudFormationStackPollerContext stackPollerContext = new CloudFormationStackPollerContext(cloudFormationClient, UPDATE_COMPLETE, errorStatuses, stack);
+        try {
+            stackPollingService.pollWithTimeout(new CloudFormationStackStatusChecker(), stackPollerContext, POLLING_INTERVAL, INFINITE_ATTEMPTS);
+        } catch (CloudFormationStackException e) {
+            LOGGER.error(String.format("Failed to update CloudFormation stack: %s", stackId), e);
+            return false;
```

Comment:
Is it ok that the exception message is only logged and doesn't get back to the user in a notification?